htdocs/scss/components/quick-reply.scss

```diff
@@ -102,16 +102,8 @@ $icon-control-height: 2.2em;
         align-items: flex-start;
     }
 
-    span#quotebuttonspan {
-        // This only exists to get something appended to it by jQuery. Its child
-        // input is the real flex item.
-        display: contents;
-
-        // Separate the "submit" buttons from the "misc" ones, if there's space.
-        // Quote is the first "misc" button.
-        input {
-            margin-left: auto;
-        }
+    #comment-text-quote {
```

Comment:
This id is anticipating a bunch of other markup helper buttons. Comment-text-em, -link, etc.